docs/source/gateway/index.rst

```diff
@@ -623,18 +630,91 @@ Once the MLflow AI Gateway server has been configured and started, it is ready t
+|                               |                |          |               | tokens and return the completion.                     |
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+
+Chat
+~~~~
+
+The standard parameters for completions routes with type ``llm/v1/chat`` are:
+
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+| Query Parameter               | Type           | Required | Default       | Description                                           |
++===============================+================+==========+===============+=======================================================+
+| **messages**                  | array[message] | Yes      | N/A           | A list of messages in a conversation from which to    |
+|                               |                |          |               | a new message (chat completion). For information      |
+|                               |                |          |               | about the message structure, see                      |
+|                               |                |          |               | :ref:`chat_message_structure`.                        |
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+| **candidate_count**           | integer        | No       | 1             | The number of chat completions to generate for the    |
+|                               |                |          |               | specified prompt, between 1 and 5.                    |
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+| **temperature**               | float          | No       | 0.0           | The sampling temperature to use, between 0 and 1.     |
+|                               |                |          |               | Higher values will make the output more random, and   |
+|                               |                |          |               | lower values will make the output more deterministic. |
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+| **max_tokens**                | integer        | No       | infinity      | The maximum completion length, between 1 and infinity |
+|                               |                |          |               | (unlimited).                                          |
++-------------------------------+----------------+----------+---------------+-------------------------------------------------------+
+| **stop**                      | array[string]  | No       | []            | Sequences where the model should stop generating      |
```

Comment:
The default stop value is None. An empty array is invalid for OpenAI and results in a bad request.

Reply:
```suggestion
| **stop**                      | array[string]  | No       | None            | Sequences where the model should stop generating      |
```